.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,50 @@
+name: Demos
+
+on:
+  pull_request:
+    branches: '**'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.8"
+    - name: python -m pip install --upgrade pip && python -m pip install numpy
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install numpy
```

Comment:
do we need specific numpy version here e.g. like for openvino python?

Reply:
No. The compilation doesn't care